tools.yml

```diff
@@ -1053,7 +1053,9 @@ tools:
   toolshed.g2.bx.psu.edu/repos/iuc/anndata_export/anndata_export/.*:
     cores: 4
   toolshed.g2.bx.psu.edu/repos/iuc/anndata_import/anndata_import/.*:
-    mem: input_size * 10
+    rules:
+    - id: anndata_import_mem_rule
+      mem: input_size * 10
```

Comment:
I disagree with this change... What is the point of doing what you can do in one line using three lines?